internal/openmetrics-exporter/alerts_collector.go

```diff
@@ -24,15 +24,15 @@ func (c *AlertsCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 	al := make(map[string]float64)
 	for _, alert := range alerts.Items {
-		al[fmt.Sprintf("%s,%s", alert.ComponentType, alert.Severity)] += 1
+		al[fmt.Sprintf("%s,%s", alert.ComponentType, alert.Severity, alert.Created, alert.Name, alert.Code, alert.Summary, alert.Issue)] += 1
```

Comment:
```suggestion
		al[fmt.Sprintf("%s,%s,%s,%s,%s,%s,%s", alert.ComponentType, alert.Severity, strconv.FormatInt(int64(alert.Created), 10), alert.Name, strconv.FormatInt(int64(alert.Code), 10), alert.Summary, alert.Issue)] += 1
```

This 'works' but is reallllly dirty. will have to check a bunch of things to see if we can get around going from int -> int64 -> string

if using this also import "strconv" in line 6